homeassistant/components/lupusec/strings.json

```diff
@@ -0,0 +1,40 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Lupus Electronics LUPUSEC connection",
+        "data": {
+          "host": "Hostname/IP",
+          "username": "Username",
+          "password": "Password",
```

Comment:
You can use translation references. Checkout Sonarr for an example

Reply:
Changed to the appropriate translation references.